router/pump.go

```diff
@@ -86,9 +86,29 @@ func ignoreContainer(container *docker.Container) bool {
 			return true
 		}
 	}
-	excludeLabel := getopt("EXCLUDE_LABEL", "")
+
```

Comment:
all changes to this file's logic can be simplified like:

```
excludeString := getopt("EXCLUDE_LABEL", "")
excludeValue := "true"
// support EXCLUDE_LABEL having a custom label value
if len(strings.Split(excludeString, ":")) == 2 {
  excludeValue = excludeStringArr[1]
}

if value, ok := container.Config.Labels[excludeLabel]; ok {
  return len(excludeLabel) > 0 && strings.ToLower(value) == strings.ToLower(excludeValue)
}
return false
```

Reply:
thanks for the feedback.. i have updated the PR.

Reply:
> thanks for the feedback.. i have updated the PR.

the update i see only added comments... https://github.com/ibrokethecloud/logspout/commit/2184712c11a6a1c0895afbc77effba03361a3de2